packit/api.py

```diff
@@ -296,6 +296,8 @@ def common_env(self, version: Optional[str] = None) -> dict[str, str]:
             if isinstance(repo, PackitRepositoryBase)
             and repo._local_project is not None
         }
+        if isinstance(self.up, NonGitUpstream):
+            env.update({"PACKIT_ACTIONS_DIR": str(self.up.working_dir)})
```

Comment:
Is this already “set in stone”? IMO we could probably go with something like `PACKIT_PWD` and use it even when we have git upstream… It would be more explicit than having it in the docs

Reply:
no, open for change, I like the idea :+1: